omod-2.2/src/test/resources/conditionTestDataset.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Remember to add License header to this file too

Reply:
Noted. Thanks.